docs/pages/includes/config-reference/okta-service.yaml

```diff
@@ -9,3 +9,27 @@ okta_service:
 
   # Points to a file with the API token as its contents on the local filesystem.
   api_token_path: /path/to/api/token
+
+  # General synchronization settings.
+  sync:
```

Comment:
Let's remove this section from the docs as we want to deprecate self-hosted Okta service.